tdrs-backend/tdpservice/scheduling/management/db_backup.py

```diff
@@ -20,7 +20,10 @@
 
 
 OS_ENV = os.environ
-content_type = ContentType.objects.get_for_model(LogEntry)
+
+def get_content_type():
+    """Get content type for log entry."""
+    return ContentType.objects.get_for_model(LogEntry)
```

Comment:
Was this a result of a linting issue or bad practice for using globals? Or is there some variations among the log lines which may result in a different content_type needing a more dynamic call/evaluation?

Reply:
This was actually causing pytest to fail because it was making a query to the DB and it wasnt marked with `@pytest.mark.django_db`. The reason we were getting that error is because the globals were being executed upon import of that module. So I had to wrap it in the "getter" so that it was executed in the correct context.